### PR TITLE
Towards ##81 Update READMEs to reflect oceanpy#1095 changes (streamlined READMEs) 

### PR DESCRIPTION
**Pull request by jfdelgad**
Towards #81 .

Changes proposed in this PR:

- Fixes broken links in main3
- Fixes example code in main3
- Fixes helpers.md

### CODE REVIEW

File: challenges/main3.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ In the same Python console:
 ```python
 from web3.main import Web3
 to_address="0xA54ABd42b11B7C97538CAD7C6A2820419ddF703E" #official judges address
-datatoken.mint(to_address, Web3.toWei(10, "ether"), {"from": alice_wallet})
+datatoken.mint(to_address, Web3.toWei(10, "ether"), {"from": alice})
 ```
 
 Finally, ensure you've filled in your Questbook entry.
```
Comment (AnaLoznianu) [inline challenges/main3.md:216]: Here, we can drop a link to desights as we will not use questbook anymore

**Comment by AnaLoznianu** [comment]
@jfdelgad I still have to test out the code, but I am confused by one thing. Why are we giving instructions to people to use Mumbai and not Polygon mainnet?

File: challenges/main3.md [hunk 2]
```diff
@@ -68,18 +68,18 @@ To share tamper-proof predictions, you'll use Arweave. You have two options, A a
   - Prepare by: get AR via [a faucet](https://faucet.arweave.net/) or [buying some](https://www.google.com/search?q=buy+arweave+tokens)
```
Comment (AnaLoznianu) [inline challenges/main3.md:68]: In option A, I think we can direct them to Arweave directly. We have a tutorial here: https://docs.oceanprotocol.com/using-ocean-market/asset-hosting#arweave

Comment (AnaLoznianu) [inline challenges/main3.md:68]: On the line with: "- Winners announced: within one week" - maybe we can give a link to the last announcement: https://blog.oceanprotocol.com/introducing-the-winners-of-the-eth-price-prediction-data-challenge-edition-2-6acdccb9271. So they know where results will be posted.
Please treat this as a suggestion

File: challenges/main3.md [hunk 3]
```diff
@@ -230,8 +230,8 @@ Load helper functions: Go to [helpers.md](../support/helpers.md) and follow the
 In the same Python console:
 ```python
 # setup
-ocean = create_ocean_instance()
-alice_wallet = create_alice_wallet(ocean) #you're Alice
+ocean = create_ocean_instance("polygon-main") # change the network name if needed
```
Comment (AnaLoznianu) [inline challenges/main3.md:233]: If we direct participants towards mumbai(polygon-test) then the judges should work on the same network

**Review by AnaLoznianu — CHANGES_REQUESTED** [review]

File: challenges/main3.md (hunk 2 above)
Comment (jfdelgad) [reply challenges/main3.md:68]: added the link to the tutorial.

**Comment by jfdelgad** [comment]
> @jfdelgad I still have to test out the code, but I am confused by one thing. Why are we giving instructions to use Mumbai and not Polygon mainnet? In the end, we want the predictions available on polygon right?

Mumbai was selected to avoid issues presented in the previous challenge, that has no access to buy matic

**Review by AnaLoznianu — APPROVED** [review]

**Pull request merged**